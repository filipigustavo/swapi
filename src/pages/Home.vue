<template>
  <Layout>
    <Search />
    <FilterBy />
    <PeopleList />
  </Layout>
</template>

<script>
import Layout from "@/components/Layout";
import Search from "@/components/Search";
import FilterBy from "@/components/FilterBy";
import PeopleList from "@/components/PeopleList";

export default {
  components: {
    Layout,
    Search,
    FilterBy,
    PeopleList
  }
};
</script>
