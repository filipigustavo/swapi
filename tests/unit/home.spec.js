import { shallowMount } from '@vue/test-utils'
import Home from '@/pages/Home'

describe('Home', () => {
  it('renders Home page', () => {
    const wrapper = shallowMount(Home)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
