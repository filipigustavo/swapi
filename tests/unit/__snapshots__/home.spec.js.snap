// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders Home page 1`] = `
<layout-stub>
  <search-stub></search-stub>
  <filterby-stub></filterby-stub>
  <peoplelist-stub></peoplelist-stub>
</layout-stub>
`;
